docs/proposed/psa-driver-interface.md

```diff
@@ -514,10 +514,6 @@ In a multithreaded environment, the driver may only call these two functions fro
 
 ## How to use drivers from an application
 
-### Declaring which cryptographic mechanism an application needs
```

Comment:
I think "Declaring which cryptographic mechanism an application needs" is still a very relevant question that should be part of this spec work. Even if the spec itself is not changing anything and pushing out changing it to a later time, it's a good opportunity to define how exactly the current system is supposed to work, since different people seem to have different understandings (ref discussions on 3313).

Reply:
It's very relevant and needs to be done, but I don't want to hold up the PSA driver spec on this, and I'm not sure if we'll end up codifying this at the PSA level rather than doing something in Mbed TLS. Drivers are mostly a new area whereas selecting which parts of the library to include involves changing what already exists in Mbed TLS.